src/web/app/components/sessions-recycle-bin-table/sessions-recycle-bin-table.component.scss

```diff
@@ -47,3 +47,7 @@
 .padding-left-7px {
   padding-left: 7px;
 }
+
+#deleted-sessions-table {
+  min-width: 700px;
```

Comment:
Maybe we can move this attribute to the `.recycle-bin-table` class in line 5? so that we can reuse the CSS classes